source/devices/AM62PX/linux/Release_Specific_Release_Notes.rst

```diff
@@ -50,6 +50,8 @@ What's new
   - Important Bug Fixes on top of Processor SDK 10.00.07.04 Release
   - RT Kernel : Real-Time Linux Interrupt Latency numbers here - :ref:`RT Interrupt Latencies <RT-linux-performance>`
   - Power Management: DeepSleep, MCU-Only and Partial IO mode - :ref:`Power Management Overview <Power-Management>`
+  - TI Apps Launcher: New Wi-Fi and Terminal Demo - :ref:`TI Apps Launcher <TI-Apps-Launcher-User-Guide-label>`
```

Comment:
I do not see this update in AM62A. Is this intentional?

Reply:
@AashvijShenai TI Apps Launcher is not packaged in AM62A image as of now.